core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -75,7 +75,7 @@ class LogManager(logDirs: Seq[File],
   // from one log directory to another log directory on the same broker. The directory of the future log will be renamed
   // to replace the current log of the partition after the future log catches up with the current log
   private val futureLogs = new Pool[TopicPartition, Log]()
-  private val logsToBeDeleted = new LinkedBlockingQueue[Log]()
+  private val logsToBeDeleted = new LinkedBlockingQueue[(Log, Long)]()
```

Comment:
Could we add a comment on what's the long value in the queue?

Reply:
Sure. It is fixed now.